### DISCUSSION TITLE
Style color of v-list-item prepend/append icon

### DISCUSSION THREAD

**Discussion by frederikheld** [vuetifyjs/vuetify#19886] [Q&A]
For whatever reason, a list item's prepend and append icon are some greyish mute color while the text of the list item is pitch black. I would like to style the icons in the same color as the text. Also I DO NOT want to have a background color for the list item, so all the variants and active state don't work.

I tried to style the icons via CSS but none of my attempts had the effect I am expecting. Why is this so hard? It should be one line of CSS.

I doesn't even seem to be possible if I use `<template #prepend>` with an `<v-icon>` inside instead of the `prepend-icon` prop.

Can anyone please give me a hint?

**Comment by frederikheld** [comment]
@johnleider I just came across [this discussion](https://github.com/vuetifyjs/vuetify/issues/9285#issuecomment-546280583) where someone complained in 2019 that there are too many options to style the list items which makes the styling obscure. Now with `base-color` added we have even more options and it became even more obscure. I see a lot of magic going on with the coloring which is fine if you are okay with the out-of-the-box experience but makes individual styling borderline impossible.

I would appreciate if we could get a mode for lists that removes all that magic and just keeps the layout and interaction mechanics of a list.

**Reply by johnleider** [reply]
Can you show me what you're trying to create?

**Reply by frederikheld** [reply]
Thank you for reaching out John :-)

![image](https://github.com/vuetifyjs/vuetify/assets/11460463/27fb9d79-0c26-46e0-8c98-9bef4c7eedd8)

I'm using the v-list, v-list-item and v-list-group as layout elements and to provide interaction like the ripple feedback when the element was clicked and the expand feature. But I don't want it to look too much like an actual list because it should visually integrate with the rest of the view. Additionally I would like to color the icons differently. The left side should be black as the text, not only in "active" state because there should be no visual difference between active and inactive in the way I use it here.

The list components are just too opinionated about how the colors work, which makes sense if you use them as a traditional list (like in the menu drawer) but limits all other use cases that require similar mechanics. I would be happy to just deep select the CSS classes but I found no way to overwrite icon styling and other stuff (like making the list component background transparent).
